CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 13.0.0
+
+* Deprecate `nbsala`
+* Deprecate `tva_ent`
+* Low impact changes since deprecating unused variables
```

Comment:
When I read this, I understand that you deprecated two variables **and** did other low impact changes.

I'm not even sure the third point is necessary at all. If you think it is important, please make sure to explain that is it not a change, but additional info about the two first lines. 

For instance you can not make it a bullet, and instead having a full sentence "These changes are low impact since the two deprecated variables were not used"